plugins/module_utils/nae.py

```diff
@@ -460,9 +460,12 @@ def get_pre_change_result(self):
             result = json.loads(resp.read())['value']['data']
         count = 0
         for x in result:
+            suppressed_event_list = self.params.get('ignore_sm')
             if int(x['count']) > 0:
                 if str(x['epoch2_details']['severity']) == "EVENT_SEVERITY_INFO":
                     continue
+                elif str(x['epoch2_details']['mnemonic']) in suppressed_event_list:
```

Comment:
would it be better if we use

if str(x['epoch2_details']['severity']) == "EVENT_SEVERITY_INFO" or str(x['epoch2_details']['mnemonic']) in suppressed_event_list:
                continue